### DISCUSSION TITLE
Use of investpy as an alternative DataSource ?

### DISCUSSION THREAD

**Discussion by xel-arjona** [asafravid/sss#12] [Ideas]
Could be nice to add [investpy](https://github.com/alvarobartt/investpy) as an alternative datafeeder given that such source covers must of world markets in an already organized ticker-asset database.

Cheers.

**Comment by asafravid** [comment]
Hi
I used to have `investpy` support in the code (you can rollback to like 6 months ago), however, investpy (i.e. investing.com) is Very strict with regards to the amount of [API] calls you make
So after ~50 the IP just gets blocked
So therefore I dropped the support for clearer code (and actual functionality..)
However, I can return it if you have some information on the amount of API calls that one can make (or is there some sleep length that can prevent IP block/blaklist?)
Keep me posted
